mods/ra2/cursors.yaml

```diff
@@ -258,11 +258,8 @@ Cursors:
 			Start: 351
 			Length: 5
 		ability:
-			Start: 68
-			Length: 5
-		ability-minimap:
-			Start: 73
```

Comment:
Is the removal of this block intentional?

Reply:
Yes, the new cursor has no minimap variant (if not defined game just uses normal one).